src/main/java/org/cardanofoundation/cfexploreraggregator/poolstatus/model/repository/PoolAggregationRepository.java

```diff
@@ -0,0 +1,13 @@
+package org.cardanofoundation.cfexploreraggregator.poolstatus.model.repository;
+
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import org.cardanofoundation.cfexploreraggregator.poolstatus.model.entity.PoolAggregationEntity;
+
+public interface PoolAggregationRepository extends JpaRepository<PoolAggregationEntity, Long> {
+
+    @Query(value = "SELECT a FROM PoolAggregationEntity a ORDER BY a.epoch DESC LIMIT 1")
+    PoolAggregationEntity getLatestPoolAggregation();
```

Comment:
this can be rewritten w/ TOP https://docs.spring.io/spring-data/jpa/reference/repositories/query-methods-details.html#repositories.limit-query-result